client/my-sites/plan-price/index.tsx

```diff
@@ -218,6 +218,16 @@ function PriceWithoutHtml( {
 	return <>{ priceObj.integer }</>;
 }
 
+export function priceAsString( price: number, currencyCode: string, isSmallestUnit = false ) {
```

Comment:
I know this already merged but I realized something: this function doesn't need to be here; it's effectively a worse version of `formatCurrency()`, the main export of the `@automattic/format-currency` package, except it doesn't put the currency symbol in the right place. (Furthermore it doesn't make a lot of sense in this file since it's not related to `PlanPrice`.)

It would be good in a follow-up to remove this. Let me know if you'd like any assistance; I'm happy to help!

Reply:
Since it's been a while and I didn't want this to get lost (lest other folks start to use this function, not knowing about the better alternative) I've removed it in https://github.com/Automattic/wp-calypso/pull/73291